Day12/src/services/resultServices.ts

```diff
@@ -0,0 +1,184 @@
+const sqlserver = require('../sqlserver');
+
+async function checkSubject(studentId :string , subjectId : string ) {
+    
+    let query = `
```

Comment:
we can use https://sequelize.org/master/ to do db operation. this give rich function that takes object.

Reply:
no need to change just go through this package and try it out in some example project